examples/more/external-project-add/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.15)
+
+project(external-project-add LANGUAGES CXX)
+
+# get external libraries
+SET(GLOBAL_OUTPUT_PATH ${CMAKE_BINARY_DIR}/3rdparty/bin/${CMAKE_BUILD_TYPE})
```

Comment:
Uppercase commands like this should be avoided (just use `set`) but I'm also not sure about this global command... Ideally, we'd like everything to be associated with a given target, I'm wondering if this could be applied at a narrower scope?

Reply:
Aggree.
This is just an old habbit.